### PR TITLE
Uninclude Adapter from Compiled Code

### PR DESCRIPTION
**Pull request by tylergraf**
We can't include the adapter in the compiled code because if the adapter is included more than once, it throws errors. So anyone who needs to use this will need to include the adapter themselves which is pretty trivial. Maybe we should add some docs for it.

## Changes
* Remove gulp task for including adapter
* Add polyfill for `formdata` (for safari and ie, although it still doesn't fix ie)
* Add conditional adapter to demo page

### CODE REVIEW

File: demo/index.html [hunk 1]
```diff
@@ -9,10 +9,14 @@
 
     <title>fs-dialog demo</title>
 
+    <script src="formdata-polyfill.js"></script>
     <script src="array-from-polyfill.js"></script>
     <script src="../../es6-promise/es6-promise.auto.js"></script>
     <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
-    <script src="../../webcomponentsjs/custom-elements-es5-adapter.js"></script>
+
+    <script>if (!window.customElements) { document.write('<!--'); }</script>
```
Comment (rosemariesadler) [inline demo/index.html]: Why is this the check we use? Don't we need the adapter every time because of compiled classes?

Comment (tylergraf) [reply demo/index.html]: We only need the adapter for browsers that support `custom-elements` v1 natively. IE Chrome and Safari.

Comment (tylergraf) [reply demo/index.html]: I copied this from what `polymer serve --compile always` spits out.

Comment (rosemariesadler) [reply demo/index.html]: Oh, that file does something different than what I thought it does. So if the polyfill for v1 elements is present (e.g. someone is running polymer2 in firefox) do we still need the adapter?

Comment (tylergraf) [reply demo/index.html]: We don't need the adapter in firefox at all.

**Comment by tylergraf** [comment]
This last commit I made updates the paths to work correctly if using the `es6` version.

File: demo/index.html (hunk 1 above)
Comment (rosemariesadler) [reply demo/index.html]: Okay, sounds good. Thanks for clarifying that for me!

**Comment by tylergraf** [comment]
Also, in updating the paths, it made the `size-limit` find the right files and we exceeded our size limit of 15KB. We were already over this, but since the `size-limit` component couldn't find some of the paths, it thought we were lower. I upped the number to `24 KB`.

File: demo/index.html (hunk 1 above)
Comment (tylergraf) [reply demo/index.html]: Ha, I just noticed I put `IE` in my comment above. I meant _i.e._

**Pull request merged**